scripts/launch_r.sh

```
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+export LD_PRELOAD=/workspaces/r-dev-env/scripts/allow_ptrace.so
+
+# If the first argument is a file, use that as R_BINARY; otherwise default.
+if [ -x "$1" ]; then
+  R_BINARY="$1"
```

Maybe throw an error in this case rather than open a version the user didn't ask for? 

But really `which_r` should handle this: as long as people have defined a valid version of R to switch to, then `which_r` should set the path correctly here.